project/Dependencies.scala

```diff
@@ -44,11 +44,10 @@ object Dependencies {
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.9.5",
 
     "org.parboiled" % "parboiled-core" % "1.4.1",
-    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-fb0c9691b")
+    excludeGuava("org.broadinstitute.dsde"       %% "rawls-model"         % "1824ef0eb")
       exclude("com.typesafe.scala-logging", "scala-logging_2.13")
       exclude("com.typesafe.akka", "akka-stream_2.13")
       exclude("com.google.code.findbugs", "jsr305")
-      exclude("bio.terra", "workspace-manager-client")
```

Comment:
@davidangb You introduced this exclusion rule a few years ago ([PR](https://github.com/broadinstitute/firecloud-orchestration/pull/861)). This is probably a long shot, but do you happen to remember the dependency conflict you were solving for? 

We pull in a couple of classes from the WSM client in the Rawls model now, so removing the exclusion rule was necessary. I'm not seeing any dependency conflicts without it, but figured I'd check with you to be safe.